app/assets/stylesheets/rides.scss

```diff
@@ -1,3 +1,75 @@
 // Place all the styles related to the Rides controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+@mixin round-button {
+  height: 120px;
+  width: 120px;
+  border-radius: 65px;
+  padding: 0px;
+  font-size: 1.3em;
+  border: 1px solid black;
+
+  @media #{$small-only} {
+    height: 100px;
+    width: 100px;
+    font-size: 1em;
+  }
+}
+
+@mixin round-badge {
+  @include round-button();
+  padding-top: 42px;
+  @media #{$small-only} { padding-top: 36px; }
+}
+
+@mixin round-link {
+  @include round-button();
+  padding-top: 46px;
+  @media #{$small-only} { padding-top: 40px; }
+}
+
+.ride {
+  @include grid-column(12);
+  @media #{$medium-only} { @include grid-column(9, $offset: 1.5); }
+  @media #{$large-up} { @include grid-column(6); }
+
+  &__body {
+    text-align: center;
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets<br>Properties should be ordered border-radius, text-align